pygerduty/requester.py

```diff
@@ -0,0 +1,34 @@
+# Requester Module used by REST API module as well as EVENTS API module.
```

Comment:
I don't have a strong preference for the module. I'd personally just have a `common` module that would share all common abstractions, e.g. The custom JSON stuff could be pulled out too.